.github/workflows/neovim-nightly.yml

```diff
@@ -0,0 +1,22 @@
+name: Neovim Nightly
+
+on: 
+  schedule:
+    # Checks daily
+    - cron: '17 11 * * *'
```

Comment:
It is!, it will be useful to know in advanced when a breaking change is introduced. And there is an example just around the corner https://github.com/johmsalas/text-case.nvim/issues/57